src/build/build.go

```diff
@@ -81,6 +68,10 @@ func checkCtx(projectCtx *project.ProjectCtx) error {
 	}
 
 	if projectCtx.BuildInDocker {
+		if projectCtx.Name == "" {
+			return fmt.Errorf("Name is missed")
```

Comment:
Is user in context what "Name" is missed?

Please add option name that is responsible for it

Reply:
The context checks are internal checks. It's an internal error (like `check_state` in the old CLI)